.chloggen/InstrumentationResourceAtt.yaml

```diff
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: instrumentation
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Limit Java and .Net auto-instrumentation resource attributes to only `Container` and `Environment` resource providers
+# One or more tracking issues related to the change
+issues: [2006]
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  This change limits the resource attributes added to Java and .Net spans and metrics to the only attributes generated by `Container` and `Environment` resource providers.
+  To enable additional resource providers, update `values.yaml` environment variable `OTEL_JAVA_ENABLED_RESOURCE_PROVIDERS` for Java, or set `OTEL_DOTNET_AUTO_{DETECTOR}_RESOURCE_DETECTOR_ENABLED` for .Net.
+  For more information on configuring these environment variables, see: https://help.splunk.com/en/splunk-observability-cloud/manage-data/available-data-sources/supported-integrations-in-splunk-observability-cloud/apm-instrumentation
```

Comment:
We should probably mention that we are doing it to avoid running into the limits